Orange/widgets/visualize/owboxplot.py

```diff
@@ -494,12 +494,13 @@ def compute_box_data(self):
             self.dist = []
             self.conts = contingency.get_contingency(
                 dataset, attr, self.group_var)
+            group_var_labels = self.group_var.values + ["Missing value"]
```

Comment:
I was thinking whether we could simplify the user experience by instead printing, say "missing thal" or "missing sepal length". Same for splitting by missing values. This would tell the user more clearly what those missing values are.

Reply:
I think it is a great idea. I added the names of the variables to the missing label.